ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/ScanConfigs/Wizard/ProfileSelection.tsx

```diff
@@ -205,12 +192,7 @@ function ProfileSelection({
                 <TableComposable>
                     <Thead noWrap>
                         <Tr>
-                            <Th
-                                select={{
-                                    onSelect: handleSelectAll,
-                                    isSelected: allRowsSelected,
-                                }}
-                            />
+                            <Th />
```

Comment:
I think you need an `aria-label` attribute, or something similar for accessibility here.